server/container_create.go

```diff
@@ -172,9 +172,11 @@ func setupContainerUser(specgen *generate.Generator, rootfs string, sc *pb.Linux
 				containerUser = userName
 			} else {
 				// Case 3: get user from image config
-				imageUser := imageConfig.Config.User
-				if imageUser != "" {
-					containerUser = imageUser
+				if imageConfig != nil {
+					imageUser := imageConfig.Config.User
+					if imageUser != "" {
+						containerUser = imageUser
```

Comment:
Thanks for checking @TomSweeneyRedHat ! well, yeah, let's follow up on this specific error message then